src/objects/Hierarchy_Settings__c.object

```diff
@@ -280,6 +280,16 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>Enable_New_Preferred_Phone_Sync__c</fullName>
+        <defaultValue>false</defaultValue>
```

Comment:
Default value is false. We enable it for new customers. 

Since the default is false, this means that it's false for unit tests as well. We should hopefully have no more issues with customer unit tests as a result of this.